app/controllers/verify/confirmations_controller.rb

```diff
@@ -11,10 +11,6 @@ def index
       finish_proofing_success
     end
 
-    def continue
-      redirect_to after_sign_in_path_for(current_user)
```

Comment:
my memory is that this route was here to provide a target for a after a user has confirmed receipt of their personal key (recovery code) after finishing the IdV process. Are we handling that personal key process differently now?